src/ex3/scripts/CliApp.mjs

```diff
@@ -0,0 +1,307 @@
+/* eslint-disable import/extensions */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable camelcase */
+import inquirer from 'inquirer';
+import { createSpinner } from 'nanospinner';
+import figlet from 'figlet';
+import gradient from 'gradient-string';
+import chalk from 'chalk';
+import asciifyImage from 'asciify-image';
+import ItemManager from './ItemManager.mjs';
+import {
```

Comment:
This is import is too long. You could import STRINGS and call each string that you need using STRINGS.XXX